src/main/java/com/arcbees/website/client/application/ApplicationView.ui.xml

```diff
@@ -75,7 +75,7 @@
                         <a href="#{nameTokens.CONTACT}" class="{res.style.menuContact}">Contact</a>
                     </li>
                     <li>
-                        <a href="#{nameTokens.DEVELOPERS}">
+                        <a href="http://dev.arcbees.com/" target="_blank">
```

Comment:
Can we remove all the developers package (All the presenters / views nametokens)? Nothing shoud point to `/developers` now?